docs/source/bricks/embedding.rst

```diff
@@ -63,7 +63,7 @@ To obtain an api key, visit: https://platform.openai.com/account/api-keys
     print(embedding_encoder.is_unit_vector(), embedding_encoder.num_of_dimensions())
 
 ``HuggingFaceEmbeddingEncoder``
---------------------------
+---------------------------------
```

Comment:
why is this a different length than the bedrock encoder below?

Reply:
Sphinx just cares that the length is longer than the text above it. Was throwing a warning before this change.

Reply:
ahh, got it